src/unum/log/log.c

```diff
@@ -45,6 +45,104 @@ void set_disabled_log_dst_mask(unsigned long mask)
     disabled_mask = mask;
 }
 
+// Flush and reset log file pointers.
+// Called when the log file changed from persistent to non-persistent
+// and viceversa.
+static void reset_log_file(LOG_CONFIG_t *lc)
```

Comment:
This won't be needed with `unum_config` prefix (see other comments).

Reply:
This is not needed with the modified code.